docs/.sphinx/wordslist.txt

```diff
@@ -1,83 +1,82 @@
 APIs
-BNF
-Baz
-CLI
-Configs
-DMI
-DNS
-DUT
-ESC
-Fi
-FooBar
-GPLv
-Github
-INI
-IQN
-Instantiation
-JSON
-Jinja
-Naur
-Parsers
-Permalink
-PlainBox
-Plainbox
-PyPI
-QML
-RTC
-SIGINT
-SRU
-Snapcraft
-Systemd
-UI
-Wi
-XSLX
 autoconf
 baz
+Baz
 bluetooth
+BNF
 boolean
 chroot
 cli
+CLI
 commandline
 config
 configs
+Configs
 cpu
 ctrl
 de
 debhelper
 defs
 desc
+DMI
+DNS
+DUT
 enquote
+ESC
+Fi
+FooBar
 formatter
 github
+Github
+GPLv
 hostname
 http
 https
+INI
 instantiation
+Instantiation
 io
 iot
+IQN
 jinja
+Jinja
 json
+JSON
 keypresses
 libre
 loopback
 lv
 mis
 miscellanea
 mkdir
+mylauncher
 myproject
 namespace
 namespaces
+Naur
 ng
 parametrize
+Parsers
 pdf
+Permalink
 plainbox
+Plainbox
```

Comment:
This looks strange too (`plainbox`, `Plainbox`, `PlainBox` all in use?)

Reply:
PlainBox has nothing to do here, will remove.

`plainbox` is used somewhere rightfully, so I'll leave it here. So is `Plainbox`.